test/TestExpectations.json

```diff
@@ -174,13 +174,6 @@
     "expectations": ["SKIP"],
     "comment": "TODO: add a comment explaining why this expectation is required (include links to issues)"
   },
-  {
-    "testIdPattern": "[waittask.spec] waittask specs Frame.waitForFunction should work when resolved right before execution context disposal",
-    "platforms": ["darwin", "linux", "win32"],
-    "parameters": [],
```

Comment:
Should it be skipped for Firefox?

Reply:
It passes there as well. Some hook is failing on CI.